### PR TITLE
Fix slack signing body decoding

### PR DESCRIPTION
**Pull request by Latent-Logic**
... Well, this is where I look the fool ...
I just tested with the merged PR #30 and found I wasn't
decoding the body of incoming requests properly.

When I originally built this I had worked with a `slack-sansio`
diff to perform the decoding inside of `validate_request_signature`
but when I moved over to writing the tests here I switched to the
upstream `slack-sansio` without the change and wrote the tests for
that incorrect behavior. This fixes the issue entirely within sirbot.

The alternative solution is to update both `slack-sansio` and this
library at the same time to match my original diff there.

### CODE REVIEW

File: sirbot/plugins/slack/endpoints.py [hunk 1]
```diff
@@ -21,7 +21,9 @@
         if slack.signing_secret:
             try:
                 validate_request_signature(
-                    await request.read(), request.headers, slack.signing_secret
+                    (await request.read()).decode("utf-8"),
```
Comment (ovv) [inline sirbot/plugins/slack/endpoints.py]: I never liked this syntax around `await` with parenthesis. in find it more readable to do that in two steps.

**Review by ovv — CHANGES_REQUESTED** [review]
No problems it happens. Good lesson for me to always do tests :D 

Thanks for catching & fixing it. Just a small nitpick and :shipit:

**Comment by Latent-Logic** [comment]
Please test as well, but as a note this code has been running for the past 24hrs talking with slack successfully.

**Comment by ovv** [comment]
Great thanks for your contribution

**Review by ovv — APPROVED** [review]

**Pull request merged**